config-connector/solutions/networking/helm/autoneg/cluster/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart to deploy example cluster configuration for GCP multi-cluster ingress
+name: gcp-mci-cluster
```

Comment:
I found that you've updated the name of this solution but there are still "mci" left in some of the names. Is it intended?

Reply:
not intended. fixed here and in other chart names.